homeassistant/const.py

```diff
@@ -69,6 +70,7 @@
 CONF_BRIGHTNESS = 'brightness'
 CONF_CODE = 'code'
 CONF_COLOR_TEMP = 'color_temp'
+CONF_COMM_TYPE = 'comm_type'
```

Comment:
Done, I moved it into components/mochad.py since it will be a shared config field for all the mochad devices.